.github/workflows/release.yml

```diff
@@ -188,3 +193,197 @@ jobs:
+          # enable Rosetta (needed only on Apple Silicon hosts)
+          sudo softwareupdate --install-rosetta --agree-to-license || true
+
+          # install Intel Homebrew into /usr/local
+          arch -x86_64 /bin/bash -c \
+            "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+          # capture the brew-env exports into GITHUB_ENV
+          arch -x86_64 /usr/local/bin/brew shellenv \
+            | sed 's/^export //g' >> $GITHUB_ENV
+
+          # alias brew via a shim
+          mkdir -p $HOME/bin
+          printf '%s\n' '#!/usr/bin/env bash' \
+            'exec arch -x86_64 /usr/local/bin/brew "$@"' \
+            > "$HOME/bin/brew"
+          chmod +x "$HOME/bin/brew"
+
+          # make sure our shim comes first
+          echo "$HOME/bin" >> $GITHUB_PATH
+
+          # verify brew is on PATH and running under x86_64
+          which brew
+          file "$(which brew)"
+          brew --version
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.24"
+
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: stable
+
+      - name: Install Colima, Lima guestagents & Docker CLI (macOS)
+        if: matrix.os == 'macos-latest'
+        run: |
+          # use the ARM-native Homebrew for host tools
+          BREW_ARM=/opt/homebrew/bin/brew
+
+          echo "Using ARM Homebrew at $BREW_ARM"
+
+          # install Lima, Colima and Docker CLI natively
+          $BREW_ARM install lima colima docker
+          $BREW_ARM install lima-additional-guestagents
+
+          # ensure ARM brews bin is on PATH for all later steps
+          echo "/opt/homebrew/bin" >> $GITHUB_PATH
+          echo "$HOME/bin" >> $GITHUB_PATH
+
+          # sanity-check
+          which colima
+          file "$(which colima)"
+          colima version
+          which docker
+          docker --version
+
+      - name: Install Docker Compose plugin (macOS)
+        if: matrix.os == 'macos-latest'
+        env:
+          GHA_ARCH: ${{ matrix.arch }}
+        run: |
+          # create plugin dir
+          mkdir -p ~/.docker/cli-plugins
+
+          # select binary according to arch
+          case "$GHA_ARCH" in
+            arm64) FILE=docker-compose-darwin-aarch64 ;;
+            x64)   FILE=docker-compose-darwin-x86_64 ;;
+            *)
+              echo "Unsupported arch: $GHA_ARCH"
+              exit 1
+              ;;
+          esac
+
+          # download the Compose V2 binary
+          curl -SL \
+            "https://github.com/docker/compose/releases/download/v2.36.2/${FILE}" \
+            -o ~/.docker/cli-plugins/docker-compose
+          chmod +x ~/.docker/cli-plugins/docker-compose
+
+          # verify
+          docker compose version
+
+      - name: Install Docker Buildx (macOS)
+        if: matrix.os == 'macos-latest'
+        run:  |
+          brew install docker-buildx
```

Comment:
specific version , same as above

Reply:
Will get these added if a breaking release happens before we get them removed 🙏